adobe_vipm/flows/migration.py

```diff
@@ -343,6 +348,36 @@ def check_running_transfers_for_product(product_id):
 
         transfer = fill_customer_data(transfer, customer)
 
+        global_sales_enabled = customer.get("globalSalesEnabled", False)
+
+        if global_sales_enabled is True:
+            gc_main_agreement = get_gc_main_agreement(
+                product_id, transfer.authorization_uk, transfer.membership_id
+            )
+            if not gc_main_agreement:
+                gc_main_agreement_data = {
+                    "membership_id": transfer.membership_id,
+                    "transfer_id": transfer.transfer_id,
+                    "customer_id": transfer.customer_id,
+                    "status": STATUS_GC_PENDING,
+                    "authorization_uk": transfer.authorization_uk,
+                }
+                try:
+                    create_gc_main_agreement(product_id, gc_main_agreement_data)
+                except AirTableHttpError as e:
+                    transfer.status = "failed"
```

Comment:
If there is any error managing the Global Customer feature, the migration should continue, and the issue should be reported to the teams. We cannot change the transfer status to fail because the Adobe account has been migrated.